modules/deployment-scripts/aks-run-helm/main.bicep

```diff
@@ -53,3 +52,8 @@ module helmAppInstalls 'br/public:deployment-scripts/aks-run-command:1.0.1' = [f
     cleanupPreference: cleanupPreference
   }
 }]
+
+output helmOutputs array = [for (app, i) in helmApps: {
```

Comment:
```suggestion
@description('Helm Command Output Values')
output helmOutputs array = [for (app, i) in helmApps: {
```

Reply:
@aristosvo , we are so close. Just need to add this description field to the new output.

Reply:
@shenglol , these changes are all set once all the checks pass. Looks like we will need you to approve as well.